### ISSUE TITLE
Font Size Increase Distorts Portal in Safari

### ISSUE THREAD

**Issue by eringeorgen** [makerdao/community-portal#439]
**Describe the bug**
Issue occurs when the type size increased and using Safari browser on a mobile device. The Text in the Callouts breaks the content frame. The same applies to the example "Status Banner" on the page.

**To Reproduce**
1. Go to https://community-portal-staging.makerfoundation.com/en/resources/content/styleguide from a mobile device, using Safari (Issue wasn't seen in Safari on desktop).
2. Enlarge the type size
3. Scroll down to the "Callouts" Heading and example Callout. The same applies to the example "Status Banner" on the page.

**Expected behavior**
The text should remain in the content frame (see example 2 other examples). The same applies to the example "Status Banner" on the page.

**Screenshots**
Safari on mobile ![IMG_4462](https://user-images.githubusercontent.com/70353047/92643216-41223b00-f296-11ea-8a8b-8837f3019668.PNG)
Firefox on mobile ![IMG_4460](https://user-images.githubusercontent.com/70353047/92643291-61ea9080-f296-11ea-8829-62cf36902166.PNG)
Safari on desktop <img width="521" alt="Screen Shot 2020-09-09 at 12 12 57 PM" src="https://user-images.githubusercontent.com/70353047/92643368-80e92280-f296-11ea-83cb-6c01f34cd96e.png">

**Smartphone (please complete the following information):**
 - Device: iPhone SE
 - Browser safari

**Comment by twblack88** [comment]
wonderful bug report @eringeorgen 👍

**Comment by MaximumCrash** [comment]
@twblack88 @eringeorgen I've labeled this won't fix.

TLDR: We're not supporting this kind of issue right now. There's a minimum limit that our designs can support. If we want to support this setting we'd need a new callout design specifically for mobile. Increasing the text size on mobile is the equivalent of hitting the zoom up hotkey on desktop, results may vary. 

I've tested and replicated this on desktop by setting the width of the screen just small enough that we start to see the site cut off. 

One culprit right now are buttons. They don't have hard rules on mobile, and trying to solve them would mean changing the spec on them. 

The primary culprit is that the phone's type size setting used above is creating a scenario where the elements at play don't work for the width of the screen. This can be seen in the screenshots used to compare the callout where the width of the of header search bar is radically different between each other. 

So how do we get around this? Well, the buttons would need to be specifically designed for mobile cases (especially the text size). 
We'd also need to redesign callouts for mobile cases like this. But it's way outside of the bandwidth for this launch. I'd say @shea-fitz should see this and consider how we can (or if we should) approach it for V2. 

The short fix for this may be my addition of #420, though this feature will be desktop focused and not mobile focused.
